<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClubEditor</class>
 <widget class="QWidget" name="ClubEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="clubNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Coach Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="coachNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="usaBtn">
       <property name="text">
        <string>USA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="canadaBtn">
       <property name="text">
        <string>Canada</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="otherCountryBtn">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="otherCountryEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address1Edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Address 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address2Edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cityEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>State:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stateCombo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Zip:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zipEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
